source/cppexpose/source/function/Function.cpp

```diff
@@ -21,16 +21,10 @@ Function::Function(const Function & other)
 
 Function::~Function()
 {
-    delete m_func;
 }
 
 Function & Function::operator=(const Function & other)
 {
-    // Delete old function
-    if (m_func) {
-        delete m_func;
-    }
-
     // Copy function
     m_func = other.m_func ? other.m_func->clone() : nullptr;
```

Comment:
All std::move does is cast lvalues to xvalues which then bind to rvalue references (and thus move assignment operators/constructors).
The result of `other.m_func->clone()` is a temporary and therefore already an xvalue, so move is not necessary here.